easybuild/tools/options.py

```diff
@@ -177,6 +178,7 @@ def override_options(self):
             'allow-modules-tool-mismatch': ("Allow mismatch of modules tool and definition of 'module' function",
                                             None, 'store_true', False),
             'cleanup-builddir': ("Cleanup build dir after successful installation.", None, 'store_true', True),
+            'compile-lmod-caches': ("Compile Lmod cache files with luac after updating them", None, 'store_true', False),
```

Comment:
i don't like the lmod option in the name. there are other module tools that can have caches